### PR TITLE
[Delta-29] Add bridge layout

### PR DESCRIPTION
**Pull request by RubicLiu**
Add bridge UI layout only.

TODO: Bridge integration

### CODE REVIEW

**Comment by deltafilzr** [comment]
Have done the merge and added this code

**Review by pjxthy — CHANGES_REQUESTED** [review]
1. Fix lint issues
2. Make sure it does not break the swap, pools, dashboard etc. logic
3. Merge

File: src/components/BurgerMenu/WalletPanel_v2.tsx [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+                      top={11}
+                      right={-5}
+                    ></Box>
```
[the remark this answers is not in the document]
Comment (deltafilzr) [reply src/components/BurgerMenu/WalletPanel_v2.tsx:188]: image

File: src/components/BurgerMenu/BridgeConnectPanel.tsx [hunk 2]
```diff
@@ -177,7 +177,7 @@ const ConnectPanel: React.FC = (props) => {
           <CloseIcon />
         </IconButton>
       </Box>
-      {!!wallet1Connected == !!wallet2Connected ? (
+      {!!wallet1Connected === !!wallet2Connected ? (
```
Comment (pjxthy) [inline src/components/BurgerMenu/BridgeConnectPanel.tsx:180]: please have better names than wallet1 & wallet2

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/components/BurgerMenu/BridgeConnectPanel.tsx (hunk 2 above)
Comment (deltafilzr) [reply src/components/BurgerMenu/BridgeConnectPanel.tsx:180]: So change to WalletOne something like this? is this one acceptable?

File: src/components/Header.tsx [hunk 3]
```diff
@@ -47,11 +48,20 @@ const Header: React.FC = (props) => {
...
   const { setMenu } = useModal();
+  const location = useLocation();
 
   useEffect(() => {
     if (isConnectedWallet) setMenu(false, "");
   }, [isConnectedWallet, setMenu]);
 
+  const handleConnect = () => {
+    if (location.pathname.substring(1) === "bridge") {
+      setMenu(true, "connectV2");
```
Comment (pjxthy) [inline src/components/Header.tsx:59]: why do we have connectV2 here?

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/components/Header.tsx (hunk 3 above)
Comment (pjxthy) [reply src/components/Header.tsx:59]: // TODO (deltafilzr): rename connectV2 to a better one
setMenu(true, "connectV2");

Comment (pjxthy) [reply src/components/Header.tsx:59]: add a child ticket to Delta-28

**Review by pjxthy — APPROVED** [review]

**Pull request merged**